scalatest/src/main/scala/org/scalatest/tools/Runner.scala

```diff
@@ -1317,7 +1318,7 @@ object Runner {
             val threadFactory =
               new ThreadFactory {
                 val defaultThreadFactory = Executors.defaultThreadFactory
-                def newThread(runnable: Runnable): Thread = {
+                def newThread(runnable: Runnable | Null): Thread = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.